src/components/v2/Layout/constants.tsx

```diff
@@ -2,45 +2,48 @@ import { isOnTestnet } from 'config';
 import { IconName } from '../Icon';
 
 interface IMenuItem {
+  /**
+   * ids have to match layout.menuItems in src/translation/translations/en.json for proper translations
+   */
+  id: string;
   href: string;
-  text: string;
   icon: IconName;
 }
 
 export const menuItems: IMenuItem[] = [
   {
+    id: 'dashboard',
     href: '/dashboard',
-    text: 'Dashboard',
```

Comment:
To elaborate on my previous comment, that's where you'd insert your text:
```typescript
text: t('layout.menuItems.dashboard'),
```

Reply:
it doesn't work with such construction. we have no texts when trying to extract translation outside of component